covid_moonshot_ml/nn/models/SchNetBind.py

```diff
@@ -11,6 +11,8 @@ class SchNetBind(SchNet):
     def __init__(self, *args, **kwargs):
         super(SchNetBind, self).__init__(*args, **kwargs)
 
+    # TODO: Does not match signature in base class
+    #  https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.models.SchNet.forward
```

Comment:
```suggestion
```
the signature of this model isn't supposed to match the base class, since we need the ligand labels in order to perform our two forward passes through the model. if there's a better way of doing this, can you please update the comment to add some details. otherwise, I think let's delete the comment to avoid confusion